app/uk/gov/hmrc/eusubsidycompliancefrontend/util/EmailTemplateHelpers.scala

```diff
@@ -25,14 +25,16 @@ import uk.gov.hmrc.eusubsidycompliancefrontend.models.Language.{English, Welsh}
 
 import java.util.Locale
 
-object TemplateHelpers {
+object EmailTemplateHelpers {
 
   private def templateIdsMap(config: Configuration, langCode: String) =  Map(
     s"createUndertaking$langCode" -> config.get[String](s"email-send.create-undertaking-template-$langCode"),
```

Comment:
I don't think you need the langcode in the keys, since we just add it onto the base name on the right.

Reply:
I need the lang code in keys to differentiate between english and welsh  template

Reply:
ok, got your point 👍